server/routes/bookingJourney/selectAdditionalSupport.test.ts

```diff
@@ -0,0 +1,169 @@
+import type { Express } from 'express'
+import request from 'supertest'
+import * as cheerio from 'cheerio'
+import { SessionData } from 'express-session'
+import { FieldValidationError } from 'express-validator'
+import { appWithAllRoutes, flashProvider } from '../testutils/appSetup'
+import { createMockBookerService, createMockPrisonService } from '../../services/testutils/mocks'
+import TestData from '../testutils/testData'
+import { FlashData } from '../../@types/bapv'
+
+let app: Express
+
+const bookerService = createMockBookerService()
+const prisonService = createMockPrisonService()
+let sessionData: SessionData
+
+const url = '/book-a-visit/select-additional-support'
+
+const bookerReference = TestData.bookerReference().value
+const prisoner = TestData.prisonerInfoDto()
+const prison = TestData.prisonDto()
+const visitors = [
```

Comment:
```suggestion
const visitors = [TestData.visitorInfoDto()]
```
Probably only need this very minimal `visitors` test data - will make it less cluttered.